### PR TITLE
Integrate blog's MVC upgrade guide

### PR DESCRIPTION
**Pull request by diplopito**
Insert all MVC changes detailed in https://docs.phalconphp.com/4.0/th-th/upgrade plus the replacement of `Phalcon\Mvc\User\Component` by `Phalcon\Plugin`

### CODE REVIEW

File: en/upgrade.md [hunk 1]
```diff
@@ -421,6 +421,91 @@ $criteria->limit(10, null);
...
+- Removed `Phalcon\Mvc\User\Component` - replaced by `Phalcon\Plugin`
+
+### Mvc\View
+- Removed `getParams`
+
+
+<hr/>
+
+## Plugin
+- Added `Phalcon\Plugin` - replaces `Phalcon\Mvc\User\Component`
```
Comment (niden) [inline en/upgrade.md]: - Added `Phalcon\Plugin` - replaces `Phalcon\Mvc\User\Component`, `Phalcon\Mvc\User\Plugin` and `Phalcon\Mvc\User\Module`

**Review by niden — CHANGES_REQUESTED** [review]

File: en/upgrade.md (hunk 1 above)
Comment (diplopito) [reply en/upgrade.md]: Very well!

**Review by diplopito — COMMENTED** [review]
Added replaced components

**Comment by niden** [comment]
Thank you @diplopito

**Pull request merged**